CI/docker-compose.yml

```diff
@@ -96,14 +90,14 @@ services:
       - TWEEKGATEWAY_SECURITY_CORS_ALLOWEDHEADERS=null
       - TWEEKGATEWAY_SECURITY_CORS_ALLOWCREDENTIALS=true
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOENDPOINT=minio:9000
-      - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOBUCKETNAME=tweek-casbin-policies
+      - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOBUCKETNAME=tweek-bucket
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOACCESSKEY=AKIAIOSFODNN7EXAMPLE
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOSECRETKEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOSECURE=false
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_MINIOPOLICYOBJECTNAME=policy.csv
       - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_NATSENDPOINT=nats://nats:4222
-      - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_NATSSUBJECT=casbin-policy-subject
-      - TWEEKGATEWAY_SECURITY_TWEEKSECRETKEYPATH=/gitkeys/tweekgit
+      - TWEEKGATEWAY_SECURITY_POLICYSTORAGE_NATSSUBJECT=version
```

Comment:
it should be hardcoded.

Reply:
Yet, we should have support for nats prefix so we can run on external NAT or run several Tweek on the same NAT